### PR TITLE
fix(license): fix upgrade button redirection

### PR DESCRIPTION
**Pull request by neolitec**
SCFRCA-1212 - update a windows licence - the redirection goes to the home page

### CODE REVIEW

File: client/app/license/redirection/upgrade/license-redirection-upgrade.controller.js [hunk 1]
```diff
@@ -1,12 +1,21 @@
-angular.module("Module.license").controller("ConfirmUpgradeRedirectionCtrl", ($location, $scope, $timeout) => {
-    "use strict";
+angular.module("Module.license").controller("ConfirmUpgradeRedirectionCtrl", class ConfirmUpgradeRedirectionCtrl {
```
Comment (antleblanc) [inline client/app/license/redirection/upgrade/license-redirection-upgrade.controller.js]: Could you also rename controller like: `ConfirmUpgradeRedirectionCtrl` => `LicenseRedirectionUpgradeCtrl`?

File: client/app/license/redirection/upgrade/license-redirection-upgrade.controller.js [hunk 2]
```diff
@@ -1,12 +1,21 @@
...
+        this.$scope.redirectToUpgrade = this.redirectToUpgrade.bind(this);
+    }
 
-    $scope.redirectToUpgrade = function () {
-        $scope.resetAction();
-        $timeout(() => {
-            $location.path(`/configuration/license/upgrade/${licenseId}`);
+    redirectToUpgrade () {
+        this.$scope.resetAction();
+        this.$timeout(() => {
```
Comment (frenautvh) [inline client/app/license/redirection/upgrade/license-redirection-upgrade.controller.js]: why 300 ms timeout ?

Comment (FredericEspiau) [reply client/app/license/redirection/upgrade/license-redirection-upgrade.controller.js]: why `$timeout` ?

Comment (neolitec) [reply client/app/license/redirection/upgrade/license-redirection-upgrade.controller.js]: Don't look at me as if I was guilty :p

Comment (FredericEspiau) [reply client/app/license/redirection/upgrade/license-redirection-upgrade.controller.js]: You could have just removed it when you reached it 😃

But we know you didn't write it down to begin with !

Comment (neolitec) [reply client/app/license/redirection/upgrade/license-redirection-upgrade.controller.js]: The timeout prevents the modal backdrop to stay here on the next page. :(

Comment (antleblanc) [reply client/app/license/redirection/upgrade/license-redirection-upgrade.controller.js]: Ok! So let's replace with a stateless modal in a separate pull request.

**Pull request merged**